src/aura/STG_CMP_Addr/STG_CMP_AddrHelper.js

```diff
@@ -27,18 +27,4 @@
 		});
 		$A.enqueueAction(runBatchAction);
     },
-    handlePhoneSync : function (component, event) {
-        var enablePrefPhone = event.getSource().get("v.value");
-        
-        this.toggleDisablePhoneEnforcementCheckbox(component, enablePrefPhone);
-        
-        $A.get('e.force:refreshView').fire();
-	},
-	toggleDisablePhoneEnforcementCheckbox : function (component, enablePrefPhone) {
```

Comment:
This method is also called in the STG_CMP_AddrController.js - I think we need to update the controller to no longer call this. I'm surprised it wasn't throwing an error when I was testing this!

Reply:
Strange! I would have expected an error as well.